### PR TITLE
Allow for A = C / B when C is generated by A * B = C

### PR DESCRIPTION
**Pull request by chrisjohgorman**
- remove doctest SKIP_IF sympy > 1.5.1
- replace xtests with test and remove reference to issue
- use solve command that oscarbenjamin suggested from upstream to fix
  issue.
- fixes issue #1079
- please use extra care with this fix as I am sick and may have made a
  mistake.

### CODE REVIEW

File: inst/@sym/mrdivide.m [hunk 1]
```diff
@@ -80,17 +77,21 @@
...
-  % not working as of 2016-01).  We don't disallow this but its not a
-  % good thing to encourage for working in Octave (since it won't work
-  % with doubles).
+  z = pycall_sympy__ (cmd, sym(x), sym(y));
 
 end
 
 
-%!test
+%!xtest
```
Comment (cbm755) [inline inst/@sym/mrdivide.m:94]: this doesn't look right?

**Review by cbm755 — COMMENTED** [review]
Looks great!  There was one test -> xtest change that I didn't understand, maybe a typo?

Do we need to do something like this for mldivide too?  (could be a different PR or maybe you can file an issue so we don't forget)

**Comment by chrisjohgorman** [comment]
I thank you for catching my mistake.  I have pushed the changes and
expected to have a chance to comment when issuing a pull request, but I
think the pull request was automatically updated.

I don't think we need to make changes to ***@***.***/mldivide.m, but I will
open an issue just in case.

**Comment by chrisjohgorman** [comment]
I just found a use case that will cause this to fail.  If B is
non-invertible, then the division will fail.

A = [sym(-1) sym(3)/2; sym(1) sym(-1)]
B = [sym(-1) sym(3)/2; sym(2)/3 sym(-1)]
C = A*B
A2 = C/B

I don't think there is a solution to this, and it fails with the old
code and sympy-1.5.1 too.  FYI.

**Comment by cbm755** [comment]
In theory some kind of minimal norm solution would be best in non-invertible situations but I suspect we should just do whatever upstream solve does for now.  Maybe you could file another issue with your non-invertible exam and we can consider there (?)

For now, I will merge this.

**Pull request merged**